### ISSUE TITLE
[Perspectives - Contrast]

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-showcase-examples#274]
What Is... title is a bit awkward, but not sure how to change due to consistency. Content of this section is seems quite jargony and unnecessarily qualified, perhaps stick to the simple statement of what it is, one simple sentence along the line of:

Onscreen text - including text displayed on images, icons, and buttons - is more easily read by most people when it is presented with sufficient contrast between the text and the background color.

**Comment by nitedog** [comment]
1. On title, would it help to put "Colors with Good Contrast" in quotations to denote that it is a single title? We would need to do that across all pages. Like: What is "X.Y.Z"?
2. Will take a pass at the paragraph (also considering the pull requests). But your suggested sentence seems long and hard to parse (at least for me).
